markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
 This week, James Fellows Yates ([@jfy133](https://github.com/jfy133)) will give a basic introduction to the vector-based graphic design tool [Inkscape](https://inkscape.org/), and then how to use the tool to create the popular 'tube map'-style pipeline diagrams.
 
 ![nf-core/eager pipeline diagram in the style of a tube-map](https://raw.githubusercontent.com/nf-core/eager/master/docs/images/usage/eager2_metromap_complex.png)
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: The content has been edited for reader-friendliness**
+
+[0:01](https://www.youtube.com/watch?v=0vKhfedYKGo&t=1)
+(host) Hello everyone, I'm here today with James Fellows Yates and he's going to talk about Inkscape and pipeline diagrams.
+
+Thank you very much. This is a repeat of a talk I gave back at the last nf-core hackathon and what I'll be talking about is how you can make pretty diagrams aimed at improving your documentation. This can be making it more eye-catchy, more accessible and making people understand better how to run your pipeline and how to interpret it. What I'll do is give a very brief introduction to what you can generate with a tool called Inkscape. Then I'll give a live demo of how you use Inkscape and the main components and functionality that I use to make and the different pipeline map constructions, which become semi-famous, at least on Twitter. Then I'll show you how I will construct a pipeline diagram such as this.
+
+[1:08](https://www.youtube.com/watch?v=0vKhfedYKGo&t=68)
+The tool that we're talking about today is Inkscape, which is an open source, pretty close to professional quality software, which can run on Linux OSx and Windows, which is really nice. It's equivalent to to Adobe Illustrator, but it is but open source, so it's free and accessible to everybody. It's a really nice and quite polished tool and I really enjoy it a lot and I've been using it for quite a few years now. The reason why we want to use Inkscape is primarily, because of the difference between raster and vector images. When you take a photo on your phone or with a digital camera, the images that you generate are raster images, so if you zoom in really closely, you start getting these very pixely squares of different colors.
```

Comment:
```suggestion
The tool that we're talking about today is Inkscape, which is an open source, pretty close to professional quality software, which can run on Linux, OSX, and Windows, which is really nice. It's equivalent to to Adobe Illustrator, but it is but open source, so it's free and accessible to everybody. It's a really nice and quite polished tool and I really enjoy it a lot and I've been using it for quite a few years now. The reason why we want to use Inkscape is primarily, because of the difference between raster and vector images. When you take a photo on your phone or with a digital camera, the images that you generate are raster images, so if you zoom in really closely, you start getting these very pixely squares of different colors.
```